<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherMonitor</class>
 <widget class="QWidget" name="WeatherMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lat_label">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Широта: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lat_input">
     <property name="placeholderText">
      <string>Введите широту</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lon_label">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Долгота:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lon_input">
     <property name="placeholderText">
      <string>Введите долготу</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="delay_label">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Задержка (сек):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="delay_input">
     <property name="placeholderText">
      <string>Введите время задержки</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="weather_label">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Информация о погоде:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="weather_info"/>
   </item>
   <item>
    <widget class="QPushButton" name="control_button">
     <property name="text">
      <string>Старт</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
